README.md

```diff
@@ -1 +1,36 @@
-# ProjectReview__Rep
\ No newline at end of file
+
+## ProjectReview__Rep
+> This is a trial htmls and css file done in collaboration.
+
+
+
+## Built With
+
+- html,css
+- git and git hub
+
+
+
+
```

Comment:
please make sure that all the extra white spaces are deleted. because unnecessary space can make you look careless.